internal/olm/client.go

```diff
@@ -117,6 +120,7 @@ func (c Client) InstallVersion(ctx context.Context, version string) (*olmresourc
 		}
 	}
 
+	packageServerKey := types.NamespacedName{Namespace: namespace, Name: packageServerName}
```

Comment:
Why it was added?

Reply:
this is not new line, its a refactor. this https://github.com/operator-framework/operator-sdk/pull/2613#discussion_r386626940 has more info